client/cmd/dexc-desktop/pkg/Build-Windows.md

```diff
@@ -0,0 +1,36 @@
+## Windows build
+
+### Setting up the build environment
+
+ Download and run setup script:
+
+ ```batch
+ curl -O https://raw.githubusercontent.com/decred/dcrdex/windows-build/client/cmd/dexc-desktop/pkg/setup-windows.cmd
+ setup-windows.cmd <branch> <repoUrl>
```

Comment:
I tried on powershell and msys2 32 bit? and getting this error here:
```
Installing go...
Installing MSYS2...
Exception calling "SetEnvironmentVariable" with "3" argument(s): "Requested registry access is not
allowed."
At C:\msys64\home\joe\git\dcrdex\client\cmd\dexc-desktop\pkg\dcrdex\client\cmd\dexc-desktop\pkg\set
up-windows-stage2.ps1:38 char:50
+ ... ('Path', "$([Environment]::GetEnvironmentVariable('Path', 'Machine')) ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : SecurityException
```

Should I be running as admin?

Reply:
That seems to be it, worked in an administrator powershell.

Reply:
Right, it needs admin to be able to set env vars - needs to be added to the doco.